include/z64actor.h

```diff
@@ -161,14 +161,16 @@ typedef struct ActorShape {
 //
 #define ACTOR_FLAG_15 (1 << 15)
 
-//
-#define ACTOR_FLAG_16 (1 << 16)
+// Player automatically accepts a Talk Offer without needing to press the A button.
+// Player still has to meet all conditions to be able to receive a talk offer (for example, being in range).
+#define ACTOR_FLAG_TALK_OFFER_AUTO_ACCEPTED (1 << 16)
 
 //
```

Comment:
Ive actually made a small adjustment here: The flag is from the perspective of the actor thats offering to talk. So it makes more sense to say "talk offer auto accepted" as in "the offer is accepted automatically by the player"

@cadmic pinging cause you already approved

Reply:
that works for me